### PR TITLE
feat(update): use `tailwindcss`

### LINKED ISSUE #156: Add a template for TailwindCSS
I want to use TailwindCSS instead of SASS, but I really lack of the knowledge to set up the boilerplate. Maybe add another template for setting up Electron + React + TypeScript + Tailwind CSS?

### LINKED ISSUE #167: [feature]: use`tailwind` to layout，and remove scss
1. [x] Use `tailwindcss` instead of scss for layout.
https://github.com/electron-vite/electron-vite-react/pull/174
2. [x]  Use CSS to replace scss for layout.
https://github.com/electron-vite/electron-vite-react/pull/168

### PR DESCRIPTION
**Pull request by RSS1102**
close #156, close #167

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,40 +1,59 @@
-import { useState } from 'react'
```
Comment (caoxiemeihao) [inline src/App.tsx:1]: Use single quotes, not double quotes.

File: src/App.tsx [hunk 2]
```diff
@@ -1,40 +1,59 @@
...
 
-console.log('[App.tsx]', `Hello world from Electron ${process.versions.electron}!`)
+console.log(
+  "[App.tsx]",
+  `Hello world from Electron ${process.versions.electron}!`,
+);
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [count, setCount] = useState(0);
```
Comment (caoxiemeihao) [inline src/App.tsx:13]: No trailing semicolon is required.

File: src/components/update/Modal/index.tsx [hunk 3]
```diff
@@ -23,7 +23,7 @@ const ModalTemplate: React.FC<React.PropsWithChildren<{
   } = props
 
   return (
-    <div className='update-modal'>
+    <div className='update-modals'>
```
Comment (caoxiemeihao) [inline src/components/update/Modal/index.tsx:26]: Why `modals` ?

File: src/main.tsx [hunk 4]
```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom/client'
 import App from './App'
 import './samples/node-api'
 import './index.css'
+import "@/components/update-tailwind/tailwind.css";
```
Comment (caoxiemeihao) [inline src/main.tsx:6]: Single quotes.

File: tailwind.config.cjs [hunk 5]
```diff
@@ -0,0 +1,20 @@
+module.exports = {
```
Comment (caoxiemeihao) [inline tailwind.config.cjs:1]: It possible use `esm` ?

File: src/components/update/Modal/index.tsx (hunk 3 above)
Comment (RSS1102) [reply src/components/update/Modal/index.tsx:26]: Omissions during debugging

File: postcss.config.cjs [hunk 6]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (RSS1102) [reply postcss.config.cjs:1]: Follow the official website settings: https://tailwindcss.com/docs/configuration#creating-your-configuration-file

**Pull request merged**